{"data_mtime": 1684620724, "dep_lines": [6, 1, 2, 3, 4, 5, 8, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 11, 10, 12, 18, 19, 9, 13], "dep_prios": [5, 10, 10, 10, 10, 10, 10, 5, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 5, 5, 5, 5, 5, 10, 5], "dependencies": ["concurrent.futures", "os", "re", "sys", "time", "warnings", "numpy", "builtins", "_typeshed", "abc", "array", "concurrent", "concurrent.futures._base", "ctypes", "genericpath", "io", "mmap", "numpy.lib", "numpy.lib.shape_base", "pickle", "typing", "typing_extensions"], "hash": "6889236c112e123fd3b4a28d108dea14ba6a6c5bbf22a94548477a31326de16c", "id": "igem.server.etl.prepare", "ignore_all": true, "interface_hash": "b62750354e6c49ac48bf224b0b6a6580b0dd8b6a525ca8f150181b945547ccae", "mtime": 1684587482, "options": {"allow_redefinition": false, "allow_untyped_globals": false, "always_false": [], "always_true": [], "bazel": false, "check_untyped_defs": false, "disable_bytearray_promotion": false, "disable_error_code": [], "disable_memoryview_promotion": false, "disabled_error_codes": [], "disallow_any_decorated": false, "disallow_any_explicit": false, "disallow_any_expr": false, "disallow_any_generics": false, "disallow_any_unimported": false, "disallow_incomplete_defs": false, "disallow_subclassing_any": false, "disallow_untyped_calls": false, "disallow_untyped_decorators": false, "disallow_untyped_defs": false, "enable_error_code": [], "enabled_error_codes": [], "follow_imports": "silent", "follow_imports_for_stubs": false, "ignore_errors": false, "ignore_missing_imports": true, "implicit_optional": false, "implicit_reexport": true, "local_partial_types": false, "mypyc": false, "platform": "darwin", "plugins": [], "strict_concatenate": false, "strict_equality": false, "strict_optional": true, "warn_no_return": true, "warn_return_any": false, "warn_unreachable": false, "warn_unused_ignores": false}, "path": "/Users/andrerico/DEV/projects/igem/igem_sandbox/igem/server/etl/prepare.py", "plugin_data": null, "size": 16316, "suppressed": ["django.core.exceptions", "django.conf", "django.utils", "ge.models", "ge.utils", "pandas", "django_thread"], "version_id": "0.991"}